.travis.yml

```diff
@@ -109,7 +109,7 @@ install:
 
     echo " >> Installing pyOptSparse";
     echo "  > Cloning pyOptSparse from OpenMDAO's fork";
-    git clone https://github.com/OpenMDAO/pyoptsparse.git;
+    git clone https://github.com/mdolab/pyoptsparse.git;
```

Comment:
should remove "from OpenMDAO's fork"  from the preceding echo for both occurrences

Reply:
Fixed